### ISSUE TITLE
Can't see images if containerd is enabled in Docker Desktop

### ISSUE THREAD

**Issue by arkeane** [microsoft/vscode-docker#3707]
Type: <b>Bug</b>

Docker Extension can't see images when containerd image store beta features is enabled in Docker Desktop 

Extension version: 1.22.2
VS Code version: Code 1.73.1 (6261075646f055b99068d3688932416f2346dd3b, 2022-11-09T02:22:48.959Z)
OS version: Darwin arm64 22.1.0
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Apple M1 (8 x 24)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|2, 2, 2|
|Memory (System)|8.00GB (0.07GB free)|
|Process Argv|--crash-reporter-id 4f630fdd-2717-45dc-ac20-bc665c0ef756|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
vscorecescf:30445987
pythondataviewer:30285071
vscod805:30301674
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411:30581797
vsaa593:30376534
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
dsvsc012:30540252
azure-dev_surveyone:30548225
pyindex848:30577860
nodejswelcome1:30587005
fc301958:30595537
3biah626:30602489
gswce1:30612156
iaj6b796:30613358
dbltrim-noruby:30604474
89544117:30613380

```

</details>

**Comment by bwateratmsft** [comment]
@arkeane Do the images show if you do `docker images` in the terminal? If so this should be solved by #3586

**Comment by arkeane** [comment]
Yes I can see them, Thank you, I'll close the issue and will wait for an update
